### PR TITLE
[FCE-390] Remove metadata from track info

### PR DESCRIPTION
**Pull request by mironiasty**
## Description

Remove metadata from tracks returned from SDK

## Motivation and Context

These values should be opaque to user.

### CODE REVIEW

File: packages/react-native-client/src/hooks/usePeers.ts [hunk 1]
```diff
@@ -14,19 +14,26 @@ export type TrackType = 'Audio' | 'Video';
...
+  type: 'Audio';
+  vadStatus: VadStatus | undefined;
+};
+
+export type VideoTrack = TrackBase & {
+  type: 'Video';
   // Encoding that is currently received. Only present for remote tracks.
   encoding: TrackEncoding | null;
-  // Information about simulcast, if null simulcast is not enabled
-  simulcastConfig: SimulcastConfig | null;
```
Comment (mironiasty) [inline packages/react-native-client/src/hooks/usePeers.ts:25]: It seems that native code never set these values 🤔

File: packages/react-native-client/src/hooks/usePeers.ts [hunk 2]
```diff
@@ -68,40 +63,38 @@ export type Peer<
...
 
+function addIsActiveToTracks<MetadataType extends Metadata>(
+  peers: ReadonlyArray<Peer<MetadataType>>,
+): Peer<MetadataType>[] {
+  return peers.map((peer) => ({
+    ...peer,
+    tracks: peer.tracks.map((track) => ({
+      ...track,
+      isActive:
+        (track as { metadata?: TrackMetadata })?.metadata?.active ?? true,
```
Comment (graszka22) [inline packages/react-native-client/src/hooks/usePeers.ts:77]: does the web sdk also pass `active` and `type` in metadata?

Comment (mironiasty) [reply packages/react-native-client/src/hooks/usePeers.ts:77]: Web SDK often rely on just removing track, so it is not always needed. But @kamil-stasiak is working on changes to also remove metadata handling from part of web SDK. So at some point, we could sync mobile and web approach

**Review by karkakol — APPROVED** [review]
👍

**Pull request merged**